scripts/basic_twoword_classifier.py

```diff
@@ -26,5 +34,5 @@ def forward(self, word1, word2):
         :return: the transformed vectors after output layer
         """
         word_composed = comp_functions.concat(word1, word2, axis=1)
-        x = F.relu(self.hidden_layer(word_composed))
```

Comment:
this line should not be deleted, now the classifier has no hidden layer

Reply:
ouch, sorry